src/convert/slices.rs

```diff
@@ -174,6 +176,38 @@ if_std! {
         fn is_none(abi: &WasmSlice) -> bool { abi.ptr == 0 }
     }
 
+    impl<T, const N: usize> IntoWasmAbi for [T; N] where Box<[T]>: IntoWasmAbi<Abi = WasmSlice> {
+        type Abi = <Box<[T]> as IntoWasmAbi>::Abi;
+
+        #[inline]
+        fn into_abi(self) -> Self::Abi {
+            <Box<[T]>>::from(self).into_abi()
+        }
+    }
+
+    impl<T, const N: usize> OptionIntoWasmAbi for [T; N] where Box<[T]>: IntoWasmAbi<Abi = WasmSlice> {
+        #[inline]
+        fn none() -> WasmSlice { null_slice() }
+    }
+
+    impl<T, const N: usize> FromWasmAbi for [T; N] where Box<[T]>: FromWasmAbi<Abi = WasmSlice> {
+        type Abi = <Box<[T]> as FromWasmAbi>::Abi;
+
+        #[inline]
+        unsafe fn from_abi(js: Self::Abi) -> Self {
+            let result = <[T; N]>::try_from(<Vec<T>>::from(<Box<[T]>>::from_abi(js)));
+            return match result {
+                Ok(arr) => arr,
+                Err(_) => panic!("JS array length does not match Rust array")
```

Comment:
This is not ideal. I see a few alternatives here:
1. Accept the panic
2. Add a `T: Default` bound and fill missing values with `default()`
3. Add a `T: JsCast` bound and fill missing values with `JsValue::UNDEFINED`

Reply:
I've gone ahead and implemented alternative 2